sensor/temp-nct75.c

```diff
@@ -0,0 +1,126 @@
+/*
+ * Phoenix-RTOS
+ *
+ * NCT75 i2c Temperature Sensor driver
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Marek Bialowas
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <errno.h>
+
+#include <sys/msg.h>
+#include <posix/utils.h>
+
+#include <i2c.h>
+
+
+static const uint32_t INVALID_TEMP = -1000 * 1000u;
```

Comment:
```suggestion
static const uint32_t INVALID_TEMP = (unsigned)(-1000 * 1000);
```

Reply:
hmm, actually will change the define to `int32_t` to be consistent with return value of `getTemp`